lib/nostrum/shard/dispatch.ex

```diff
@@ -194,7 +194,7 @@ defmodule Nostrum.Shard.Dispatch do
     do: {event, UserCache.update(p), state}
 
   def handle_event(:VOICE_STATE_UPDATE = event, p, state),
-    do: {event, p, state}
+    do: {event, GuildServer.voice_state_update(p.guild_id, p), state}
```

Comment:
The documentation for this event is located [here](https://github.com/Kraigie/nostrum/blob/master/lib/nostrum/consumer.ex#L112). Could you modify this as well to better represent what is now being returned?